front/src/applications/operationalStudies/types.ts

```diff
@@ -66,6 +66,8 @@ export type ManageTrainSchedulePathProperties = {
   length: number;
   trackSectionRanges: NonNullable<PathfindingResultSuccess['track_section_ranges']>;
   incompatibleConstraints?: IncompatibleConstraints;
+  zones: PathProperties['zones'];
+  operationalPoints: SuggestedOP[];
```

Comment:
This have side effects in multiple places in operational studies (maybe that's why I can't test the PR), we should create a new type in application/stdcm/types.ts for stdcm purpose as most of the `ManageTrainSchedulePathProperties` properties aren't need here (you can verify what properties from pathProperties we use across stdcm).

```suggestion
   export type StdcmPathProperties = {
      manchetteOperationalPoints?: OperationalPoint[];
      geometry: NonNullable<PathProperties['geometry']>;
      suggestedOperationalPoints: SuggestedOP[];
      zones: NonNullable<PathProperties['zones']>;
   }
```
- `zones` are `NonNullable` because as long as you ask for them in the `pathProperties` endpoint, they will return them.
- we don't use electrifications, allWaypoints, trackSectionsRanges, length
- as we will have "pure" `suggesredOperationalPoints`, you don't need the `operationalPoints` property you just added either